analyzers/rolaguard_base_analyzer/ResourceMeter.py

```diff
@@ -81,6 +82,13 @@ def device_resource_usage(self, device, packet, policy_manager):
                         comment="The device has transmitted again",
                         commit=False
                     )
+                    emit_alert(
```

Comment:
Why there is a list of issues? How can be more than one issue for the device?

Reply:
Actually, it should never find more than one issue. But it was made that way to handle the possible situation of duplicated issues due to a bug somewhere else in the code. Should I change it?

Reply:
Ok!